aviary/docs/user_guide/pre_mission_and_mission.ipynb

```diff
@@ -96,7 +96,7 @@
    "outputs": [],
    "source": [
     "# Testing Cell\n",
-    "# ********NOTE******** #\n",
+    "# ****NOTE**** #\n",
     "# Is this supposed to be bus_variables or parameters?\n",
```

Comment:
@johnjasa @jdgratz10

Reply:
[jdgratz10](https://github.com/jdgratz10) [yesterday](https://github.com/OpenMDAO/Aviary/pull/311#discussion_r1661355988)
I believe based on the context it should be bus_variables, since this example is walking through how to use the bus. The get_parameters function is for variables that are dymos parameters during mission, while the get_bus_variables function is for variables that are indeed dymos parameters during the mission but that are also calculated during the pre-mission portion of the external subsystem. John, feel free to correct me if something has changed here, but that's the way it was back when I built the bus port initially.

Reply:
here the docs say:

>"Variables that begin with 'aircraft:' are connected when you use the get_parameters() method within SubsystemBuilderBase.\n"

however, neither of these methods appears to do that

Reply:
@johnjasa maybe you could chime in here, I don't remember the details of the builder base for this case. However, I'm pretty sure the bus is agnostic to whether or not the variables begin with 'aircraft:'.

Reply:
Jennifer, I agree with everything you're saying.

I just checked to confirm that `get_parameters()` is called in [`methods_for_level2.py` here](https://github.com/OpenMDAO/Aviary/blob/main/aviary/interface/methods_for_level2.py#L1071) and is used in `setup_trajectory_params` and then has logic to only add variables that have `':'` in the name (so `aircraft:` etc).

I think you could check both `get_parameters()` and `get_bus_variables()` here, but what "check" means is sort of amorphous. Carl, as you noted, those methods _themselves_ don't do much, but those methods are called in `methods_for_level2.py`. Maybe the check is something as simple as making sure that `SubsystemBuilderBase` has methods that are called `get_parameters()` and `get_bus_variables()` to make sure the methods don't change names. I don't think you'd need to do much more than that.

Reply:
Should I update the docs to reflect that it adds all variables with a `:` instead of variables that start with `aircraft:`?

Reply:
@Kenneth-T-Moore